src/marketing/create-new-rec.md

```diff
@@ -66,3 +66,20 @@ After you create and activate the recommendation unit, Magento will begin [colle
     - **Save as draft** to edit the recommendation unit at a later date. Note that you cannot modify the page type or recommendation type for a recommendation unit in a draft state.
 
     - **Activate** to enable the recommendation unit on your storefront.
+
+## Preview Recommendations {#preview}
+
+Use the **Recommended products preview** panel to preview what products will be displayed in a recommendation unit when deployed to your storefront.
+
+![Recommendation preview]({% link images/images-ee/prex-preview.png %}){: .zoom}
+
+|Field|Description|
+|---|---|
+|**Name**|Name of the product|
```

Comment:
We don't usually bold the terms in a field description list. Doesn't really need it.